spec/support/ruby_version_helpers.rb

```diff
@@ -0,0 +1,23 @@
+module RubyVersionHelpers
+  RUBY_2_4 = Gem::Version.new('2.4.0')
+  CURRENT_RUBY = Gem::Version.new(RUBY_VERSION)
+
+  def numeric_type
+    if CURRENT_RUBY < RUBY_2_4
+      Fixnum
+    else
+      Integer
+    end
+  end
+  module_function :numeric_type
+
+  def a_numeric
+    type = numeric_type
+    if type == Integer
+      "an #{type}"
+    else
+      "a #{type}"
+    end
+  end
+  module_function :a_numeric
```

Comment:
I made these "module functions" so that they can both be reference directly (`RubyVersionHelpers.a_numeric`), included as instance methods (`include RubyVersionHelpers`), and extended as class methods (`extend RubyVersionHelpers`).